dex-auth/dex-ldap/base/kustomization.yaml

```diff
@@ -6,3 +6,10 @@ resources:
 - namespace.yaml
 - deployment.yaml
 - service.yaml
+images:
+- name: osixia/openldap
+  newName: osixia/openldap
+  newTag: latest
```

Comment:
@yanniszark should we file an issue to pin to specific images rather than using latest?